### ISSUE TITLE
THREE.WebGLProgram: Unsupported encoding: undefined

### ISSUE THREAD

**Issue by ezewu** [mrdoob/three.js#20935]
**Describe the bug**

Console three.module.js?5a89:16731 THREE.WebGLProgram: Unsupported encoding: undefined

***Code***

```js
const TextureLod = new THREE.TextureLoader()
const mapImage = TextureLod.load('static/demo4/textures/001.jpg')
mapImage.wrapS = mapImage.wrapT = THREE.RepeatWrapping
mapImage.encoding = THREE.sRGBEncoding

const material2 = new Nodes.MeshStandardNodeMaterial()
material2.roughness = new Nodes.FloatNode(0.8)
material2.metalness = new Nodes.FloatNode(0)

const map = new Nodes.TextureNode(mapImage)
map.uv = new Nodes.UVTransformNode()
map.uv.setUvTransform(0, 0, 6, 6, 0)
material2.map = map
material2.normalMap = TextureLod.load('static/demo4/textures/normal2.png')
```

**Platform:**
 - OS: [Windows, MacOS, Linux, Android, iOS]
 - Browser: [Chrome, Firefox, Safari, Edge]

**Comment by ezewu** [comment]
Version number 0.123.0

**Comment by Mugen87** [comment]
Should be fixed via #20847.
